ocfweb/docs/docs/staff/backend/switch.md

```diff
@@ -1,15 +1,22 @@
 [[!meta title="Managed switches"]]
 
-We use an [Arista 7050SX-64][primary-switch] 10GbE switch as a primary switch
-for our servers, and two [Arista 7048T-A][secondary-switch] 1GbE switches for
+We use an [Arista 7050S-64][primary-switch] 10GbE switch as a primary switch
+for our servers, and two [Arista 7048T-4S][secondary-switch] 1GbE switches for
 desktops and management respectively. These devices were donated to us by
-Arista Networks in Fall 2018. Each device in the lab connects first to the back
+Arista Networks in Fall 2018. 
+
+The Arista 7050S-64 is a Trident+ based switching platform, featuring 48 10G-mode
+ports, and 4 40G-mode ports. Each of the SFP+ cages on the 7050S-64 can support 10G and 1G mode operation.  
+The QSFP ports can be channelized out to 4x10G in order to gain a total of 64 10G ports in dire situations.  However,
+keep in mind that changing the QSFP+ ports from 10G to 40G will cause all the interfaces to flap (including the management interface!) -- so schedule accordingly.
```

Comment:
I'm also not sure people will understand the concepts mentioned here without quite a lot more networking experience. Words like "cages," "SFP+," "QSFP," "QSFP+," etc. I believe will be confusing and might need some references or explanation. Is this doc aimed towards general OCF staff, or how much experience should one have to understand it?

Reply:
+1

this document contains much more jargon which ought to be simplified or explained, since otherwise it gets in the way of quickly finding the information we need. Repeating the technical specifications like Trident+ based platform or QSFP+ ports is probably not relevant, whereas mentioning that changing the line rate on the interface flaps all the interfaces probably is.

Reply:
These specifications are relevant.  I will spend some time elaborating these terms for new staffers; but from my viewpoint, there should be some knowledge of these concepts before manipulating the switch configuration.  After all, terms like "(Q)SFP" and "cages" should be no different than "DisplayPort", "C13/C14/IEC60320", and "HDMI."  Someone who isn't familiar with these terms, frankly, has no business modifying the switch config at all.  We have other equipment for playing around with switch configurations, and staffers should gain some experience there before managing production infrastructure.

Regarding the comment about the switch platform: The term `trident+` gives you a general idea of what to expect from a particular switch platform.  As a popular switch asic, with some networking experience, you should be able to have a general idea of what features to expect and what to keep in mind from the name.  Perhaps it is not as relevant in the context of the OCF; but it's a good-to-know.